src/app/business/common/components/searchBar/searchBar.js

```diff
@@ -0,0 +1,332 @@
+import React, {Component} from 'react';
+import keycode from 'keycode';
+import uuidv4 from 'uuid/v4';
+import decodeUriComponent from 'decode-uri-component';
+import {isEqual, noop} from 'lodash';
+import styled from '@emotion/styled';
+import Downshift from 'downshift';
+import {css} from 'emotion';
+
+import ClearIcon from '../icons/clear';
+import PropTypes from '../../../../../utils/propTypes';
+
+import {IconButton} from '../iconButton';
+import SearchInput from './overrides/searchInput';
+
+import {ice, darkSkyBlue, white} from '../../../../../../assets/css/variables/colors';
+
+const InputWrapper = styled('div')`
+    border: 1px solid ${ice};
+    background-color: ${white};
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    min-height: 40px;
+    border-radius: 20px;
+    padding: 1px;
+    position: static;
+`;
+
+const searchInputWrapper = css`
+    flex-grow: 1;
+    margin-left: 2px;
+`;
+
+const clearButton = css`
+    border: none;
+    z-index: 1;
+    outline: none;
+    margin-right: 2px;
+    
+    &:focus {
+        box-shadow: 0 0 3pt 3pt ${darkSkyBlue};
+    }
+`;
+
+// use getRootProps https://github.com/paypal/downshift#getrootprops
+const SearchInputWrapper = ({innerRef, ...rest}) => (
+    <div
+        className={searchInputWrapper}
+        ref={innerRef}
+        key={this}
```

Comment:
This looks like an addition from the code in substra-ui, could you explain why it is needed?

Reply:
I got errors that told me to add a key argument to this element.